src/MSAL.PCL/TokenCacheNotificationArgs.cs

```diff
@@ -28,33 +28,43 @@
 namespace Microsoft.Identity.Client
 {
     /// <summary>
-    /// Contains parameters used by the ADAL call accessing the cache.
+    /// 
     /// </summary>
-    public sealed class TokenCacheNotificationArgs
+    public enum NotificationType
     {
         /// <summary>
-        /// Gets the TokenCache
+        /// 
         /// </summary>
-        public TokenCache TokenCache { get; internal set; }
+        RefreshTokenCache,
+        /// <summary>
+        /// 
+        /// </summary>
+        TokenCache
```

Comment:
>TokenCache [](start = 8, length = 10)

can we explicitly say AccessTokenCache? we are separating the two cache essentially.